climt/_lib/rrtmg_lw/rrtmg_lw_rad.f90

```diff
@@ -744,8 +744,8 @@ subroutine inatm (iplon, nlay, icld, iaer, &
       taucmc(:,:) = 0.0_rb
       ciwpmc(:,:) = 0.0_rb
       clwpmc(:,:) = 0.0_rb
-      reicmc(:) = 0.0_rb
-      relqmc(:) = 0.0_rb
+      reicmc(:) = 20.0_rb
```

Comment:
Why is this needed? This input validation must happen at the python level based on the flag that is
passed in. If this was not caught in earlier version of RRTMG itself, then you should revert this change
and open an issue so that it gets fixed.

Reply:
Hi Joy,
Ah, I just ran my test case again and you're right that this isn't necessary, sorry! I was confused, because reicmc and relqmc are defined as local variables in the subroutine rrtmg_lw with an extra model layer compared to reicmcl and relqmcl, which would then have a value of 0, but it seems that the extra layer is not used.